src/extension.ts

```diff
@@ -122,7 +122,7 @@ function initializeListenersForFolder(context:ExtensionContext, folder:Workspace
 }
 
 function getRelativeNlsDocumentForPackage(packageJsonPath : Uri) : Uri {
-	return packageJsonPath.with({path: join(dirname(packageJsonPath.fsPath), "package.nls.json")});
+	return packageJsonPath.with({path: Uri.joinPath(packageJsonPath, "package.nls.json").fsPath});
```

Comment:
I dont think that this quite works, because instead of resolving `/a/b/c/package.json/../package.nls.json` it will resolve `/a/b/c/package.json/../package.nls.json` it will resolve `/a/b/c/package.json/package.nls.json`

Reply:
So then what would work?